### ISSUE TITLE
Subtle Antighost getting switched with other players

### ISSUE THREAD

**Issue by Etgfrog** [Skyrat-SS13/Skyrat-tg#3307]
Reporting client version: 513.1527

## Round ID:
736

## Testmerges:
- [[TM-ONLY] Disables jukeboxes as orderable cargo crates.](https://github.com/Skyrat-SS13/Skyrat-tg/pull/3190)
- [[Non-modular] Buffs blob.](https://github.com/Skyrat-SS13/Skyrat-tg/pull/3073)
- [Sizeplay v 0.5](https://github.com/Skyrat-SS13/Skyrat-tg/pull/3255)
- [Coronal Mass Ejection Event](https://github.com/Skyrat-SS13/Skyrat-tg/pull/3280)

## Reproduction:
I've experienced this twice now, first time there was a large amount of glow shrooms on the station, admin had nuked them because of the issue. Second time was this one, the extreme coronal mass ejection event happened, it effected nearly all the station. Time dialation was at I think 70% at the time it happened. I am unsure who it got swapped with but it happened at roughly the 1 hour mark.

**Comment by Thrakerzod** [comment]
Just had this happen as well, round ID 1648 I believe, was just doing some harmless cuddling with a friend in a Sauna, both times I tried to make subtle/anti-ghost subtle emotes, ended up getting not what I typed. Caused some awkward confusion, especially the second one which was full on ERP stuff. Dunno if anything was going on to cause some lag, but I was told if two people use the emote nearly the same time, one message gets deleted while the other goes to the other person? Second one happened roughly 10-15 minutes before the round ended, but if you would like, can message me on Discord for the specific messages to look for. Don't wanna put them here to keep the private, private. Same name there as on here.
